### DISCUSSION TITLE
Velero nodes sizing

### DISCUSSION THREAD

**Discussion by OctavioRocha** [vmware-tanzu/velero#3391] [Community support Q&A] [answered]
Hi everyone,

Can anyone say if there is an optimal sizing for Velero nodes in a Cluster, based on the number of workloads running in the Cluster?

For example:
I have a scenario in GCP with one Cluster for each Business Unit, but the number of applications running in each Cluster can vary as so the sizing of each node pool as well (number of nodes/machine type).

That being said, is there a sizing recommendations for Velero in each cluster?

Sorry if this has already discussed, I dig a little but could not find any specific related my inquiry.

Thanks in advance.

**Comment by dsu-igeek** [comment, marked as answer]
Velero runs in a single Pod unless Restic is installed in which case there is a Restic daemonset in addition to the Velero pod.  The default values for the Velero install should be adequate for most workloads.  Velero currently executes on backup per cluster at a time, so the number of workloads in the cluster is not a factor.
